langlib/lang.xml/src/main/java/org/ballerinalang/langlib/xml/ElementChildren.java

```diff
@@ -15,49 +15,50 @@
  * specific language governing permissions and limitations
  * under the License.
  */
-package org.ballerinalang.langlib.internal;
+package org.ballerinalang.langlib.xml;
 
 import org.ballerinalang.jvm.XMLNodeType;
 import org.ballerinalang.jvm.scheduling.Strand;
 import org.ballerinalang.jvm.values.XMLItem;
 import org.ballerinalang.jvm.values.XMLSequence;
 import org.ballerinalang.jvm.values.XMLValue;
-import org.ballerinalang.jvm.values.api.BXML;
 import org.ballerinalang.model.types.TypeKind;
 import org.ballerinalang.natives.annotations.Argument;
 import org.ballerinalang.natives.annotations.BallerinaFunction;
 import org.ballerinalang.natives.annotations.ReturnType;
 
-import java.util.ArrayList;
-
 /**
  * Return lift getChildren over sequences.
  *
  * @since 1.2
  */
 @BallerinaFunction(
-        orgName = "ballerina", packageName = "lang.__internal",
-        functionName = "children",
+        orgName = "ballerina", packageName = "lang.xml",
+        functionName = "elementChildren",
         args = {@Argument(name = "xmlValue", type = TypeKind.XML)},
         returnType = {@ReturnType(type = TypeKind.XML)},
         isPublic = true
 )
-public class Children {
+public class ElementChildren {
 
-    public static XMLValue children(Strand strand, XMLValue xmlVal) {
+    public static XMLValue elementChildren(Strand strand, XMLValue xmlVal, Object name) {
```

Comment:
I think we can simply do a `name == null` check here.

Reply:
+1